frontend-react/src/components/Table/TableRows.tsx

```diff
@@ -170,7 +173,7 @@ export const TableRows = ({
     }, [rows, addingNewRow, updatedRow, columns]);
 
     return (
-        <>
+        <tbody className="font-mono-2xs">
```

Comment:
Can the className be a prop since the look and feel is moving in another direction or will be different in other features i.e Message Tracker

Reply:
Alright, it will default to the same font-mono-2xs but you're able to override it with the `Table.fontClass` prop